src/vstest.console/Resources/Resources.resx

```diff
@@ -734,4 +734,7 @@
   <data name="TestSourcesDiscovered" xml:space="preserve">
     <value>A total of {0} test files matched the specified pattern.</value>
   </data>
+  <data name="InvalidTestRunParameterArgument" xml:space="preserve">
+    <value>The {0} argument is invalid.</value>
```

Comment:
Quote an example.

The test run parameter argument '{0}' is invalid. Please use the format below.
Format : TestRunParameters.Parameter(name=<name>, value=<value>)

Reply:
changed

Reply:
Why don't I see the change ?